x-pack/plugin/eql/qa/common/src/main/resources/test_queries.toml

```diff
@@ -248,16 +248,6 @@ case_insensitive = true
 query = 'process where process_name >= "SYSTE" and process_name <= "systex"'
 expected_event_ids  = [1, 2]
 
-
```

Comment:
Because for some reason it keeps running instead of being excluded and it fails due to case comparisons.
I've raised another issue to simply the current approach - copying the queries content across different files is error prone and leads to confusion on what queries get executed when...
https://github.com/elastic/elasticsearch/issues/61841